### DISCUSSION TITLE
Cannot get frequencies from trips endpoint

### DISCUSSION THREAD

**Discussion by brandonfcohen1** [transitland/transitland#381] [Q&A] [answered]
With every search I make on the `trips` endpoint, the `frequency` array is always empty. Is it just that very few agencies publish this, or is something going wrong?

Example request: 
https://transit.land/api/v2/rest/routes/12252920/trips?apikey={}&limit=100

**Comment by drewda** [comment, marked as answer]
In their GTFS feeds, transit agencies may specify the times for given trips using either "stop times" or "frequencies." On the trips were you see an empty `frequency` array, that means that the agency is using only stop times.

Yes, very few agencies use the frequencies approach. Of the thousand-plus feeds in Transitland, only ~77 feeds include the `frequencies.txt` file.

**Reply by tjhorner** [reply]
I'd love to learn how many feeds use `frequencies.txt` these days. I'm writing a GTFS consumer and evaluating whether it's worth implementing support for frequency-based trips now or waiting until we run into a feed that actually uses them. haha
